src/App.tsx

```diff
@@ -1,11 +1,28 @@
-import List from "./List.tsx";
+import React from 'react';
+
+interface ColoredTextProps {
+  text: string;
+  color: string;
+}
+
+const ColoredText: React.FC<ColoredTextProps> = ({ text, color }) => {
```

Comment:
Could you please create a separate file for the ColoredText component? This will help improve the maintainability and organization of our codebase. Thank you!

Reply:
We created a new file called Colorcounter.ts